modules/generator/processor/servicegraphs/servicegraphs.go

```diff
@@ -88,7 +88,7 @@ type Processor struct {
 	logger             log.Logger
 }
 
-func New(cfg Config, tenant string, registry registry.Registry, logger log.Logger) gen.Processor {
+func New(cfg Config, tenant string, reg registry.Registry, logger log.Logger) (gen.Processor, error) {
```

Comment:
Did something push you to add an `error` to the return signature? I think it's fine for consistency but as far as I can tell it's never set to anything else but `nil`?

Reply:
Ah yes, this might have been a detour that I didn't end up committing.  I'll back out the signature change.